app/youtube/service.go

```diff
@@ -346,6 +363,30 @@ func (s *Service) isNew(entry ytfeed.Entry, fi FeedInfo) (ok bool, err error) {
 	return true, nil
 }
 
+// isRelevant checks if entry matches all filters for the channel feed
+func (s *Service) isRelevant(entry ytfeed.Entry, fi FeedInfo) (ok bool, err error) {
```

Comment:
I'm not sure the name of the function communicating the intent well. This one is not about relevance, but rather filtering, i.e. isFiltered or isAllowed or smth like this